pkg/rancher-desktop/main/dashboardServer/index.ts

```diff
@@ -85,9 +85,7 @@ export class DashboardServer {
       .get(
         '*missing',
         (_req, res) => {
-          res.sendFile(
-            path.resolve(paths.resources, 'rancher-dashboard', 'index.html'),
-          );
+          res.sendFile('rancher-dashboard/index.html', { root: paths.resources });
```

Comment:
Should there be a comment about the hidden file issue here?

Reply:
Added. Also, the rate limit code scanner alert is harmless, because it's the local user making the requests (and they can cause high CPU usage much easier).